### PR TITLE
tests: Add separate WSI test file

### PR DESCRIPTION
**Pull request by sfricke-samsung**
Move `VkLayerTest` tests I found to a new file dedicated for WSI as more and more WSI tests are written in future, nicer to have in a single file by itself to find other tests for reference

### CODE REVIEW

**Review by ncesario-lunarg — APPROVED** [review]
LGTM!

This reminds me, at some point I'd like to break up vkpositivelayertests.cpp...>13K lines in a single file is getting a little unwieldy.

File: tests/vklayertests_wsi.cpp [hunk 1]
```diff
@@ -0,0 +1,1684 @@
+/*
+ * Copyright (c) 2015-2021 The Khronos Group Inc.
+ * Copyright (c) 2015-2021 LunarG, Inc.
+ * Copyright (c) 2015-2021 Google, Inc.
+ * Modifications Copyright (C) 2020-2021 Advanced Micro Devices, Inc. All rights reserved.
```
Comment (ncesario-lunarg) [inline tests/vklayertests_wsi.cpp:5]: @jeremyg-lunarg caught that this should be the union of the copyrights found in the 2 files being copied from, so I think
```
 * Copyright (c) 2015-2021 Valve Corporation
```
needs to be added. I'm guessing the authors from the files being copied from should also be added?

**Review by ncesario-lunarg — CHANGES_REQUESTED** [review]

File: tests/vklayertests_wsi.cpp (hunk 1 above)
Comment (sfricke-samsung) [reply tests/vklayertests_wsi.cpp:5]: ok, so not the best with this area of stuff (and hate to do something wrong by accident), I just took what was in `vklayertests_others.cpp` and the union would makes sense

Will just union the author names as well, will push up and someone can tell me if something still seems wrong with it

**Review by ncesario-lunarg — APPROVED** [review]
:+1: thanks!

**Pull request merged**